tools/influx_data_manager/influx_data_manager.xml

```diff
@@ -28,12 +37,37 @@
         <param type='data' format='tabular,txt' argument='--mmet' label='File containing specie concentrations for experiments (.mmet file)' optional='true'/>
         <param type='data' format='tabular,txt' argument='--cnstr' label='File containing constraints onf fluxes and specie concentrations for experiments (.constr file)' optional='true'/>
         <param type='data' format='tabular,txt' argument='--opt' label='File containing options for influx_si (.opt  file)' optional='true'/>
+        <param type='data' format='tabular,txt' argument='--mapping' label='Mapping of names and corresponding new names for input files' optional='true'/>
+        <conditional name='normalization'>
+            <param type='select' name='norm' label='Normalization of extracellular fluxes' help='Select "Yes" to normalize extracellular fluxes by specific value (glc uptake for example)'>
+                <option value='true'>Yes</option>
+                <option value='false'>No</option>
+            </param>
+            <when value='true'>
+                <conditional name='norm_type'>
+                    <param type='select' name='norm_type' label='Normalization type' help='Select the type of normalization to apply to fluxes'>
+                        <option value='text'>Parameter Name</option>
+                        <option value='float'>Custom Value</option>
+                    </param>
+                    <when value='text'>
+                        <param type='text' name='norm_text' label='Parameter name' help='Name of the parameter to normalize fluxes by (e.g. glc_uptake)'/>
+                    </when>
```

Comment:
I assume that only certain text values are supported, so I assume a select would be better suited here? Or is this supposed to be a column from a table?